app/models/user.rb

```diff
@@ -4,4 +4,5 @@ class User < ApplicationRecord
          :jwt_authenticatable, jwt_revocation_strategy: JwtDenylist
 
   has_many :space_cws
+  has_many :reservations
```

Comment:
Good job implementing the relationship because a user may have many reservations.